src/__tests__/to-have-style.js

```diff
@@ -30,7 +31,9 @@ describe('.toHaveStyle', () => {
     expect(container).toHaveStyle({ height: '100%' });
     expect(container).toHaveStyle({ color: 'white' });
     expect(container).toHaveStyle({ width: '50%' });
+    expect(container).toHaveStyle([[{ width: '50%' }]]);
     expect(container).toHaveStyle({ transform: [{ scale: 2 }, { rotate: '45deg' }] });
+    expect(container).toHaveStyle({ transform: [{ rotate: '45deg' }, { scale: 2 }] });
```

Comment:
The fact that these `transform` assertions all pass, seems to be incorrect. If I understand correctly the order of transformations is generally important, as e.g. `scale: 2` + `translateX: 10` will result in different object than `translateX: 10` + `scale: 2`. 

Maybe we should actually threat these other `transform` variants as counter-examples:
```suggestion
    expect(container).not.toHaveStyle({ transform: [{ rotate: '45deg' }, { scale: 2 }] });
```

@felippepuhle wdyt?

Reply:
Nice catch! I didn't notice order would matter here